### ISSUE TITLE
Error during builidng YateBTS

### ISSUE THREAD

**Issue by traghav** [Nuand/YateBTS#6]
Trying to build YateBTS on Ubuntu, getting the following errors:
`Makefile:122: recipe for target 'MSInfo.o' failed
make[2]: *** [MSInfo.o] Error 1
make[2]: Leaving directory '/home/lab/dev/sdr/yatebts/mbts/GPRS'
Makefile:140: recipe for target '../GPRS/libGPRS.a' failed
make[1]: *** [../GPRS/libGPRS.a] Error 2
make[1]: Leaving directory '/home/lab/dev/sdr/yatebts/mbts/apps'
Makefile:53: recipe for target 'all' failed
make: *** [all] Error 2
`

**Comment by elcuervo** [comment]
Having the same issue running  both ubuntu and xubuntu 17 stock.

**Comment by Zibri** [comment]
same here... solution?

**Comment by l-fy** [comment]
Please use http://forum.yate.ro

**Comment by jbarea** [comment]
I have the same exact problem when trying to build yatebts under kali linux. Is there some workaround to make it work?

**Comment by jbarea** [comment]
Anyone has builded succesfully yate-bts?

**Comment by traghav** [comment]
It was a dependency issue, after sorting it, I could build YateBTS without errors.

**Comment by jinnu92** [comment]
what is the dependency , that causing this type of error  . can u please tell me .

**Comment by traghav** [comment]
I downgraded my GCC version to compile as is. However if you comment out a couple of lines (GPRSLOG) in yatebts/mbts/GPRS/MSInfo.o (they are for logging, should not affect functionality). They should compile. Actual line numbers would be present in your make error.

**Comment by muphenmahn** [comment]
hey @traghav why don't you tell us what version you downgraded to instead of just giving out 95% of the information, this is a community not a competition "It was a dependency issue, after sorting it, I could build YateBTS without errors." that was your first response? how does that help anybody?

Patient: Sir my heart is failing
Doctor: It's a dependency issue, my heart is just fine, fuck your heart, I'll leave it up to you to find the "dependency issue" 

you cant even google that you ass.

**Comment by traghav** [comment]
Hey @muphenmahn, sounds like you are having a bad day. Sorry I made your feel this way. I downgraded from GCC 5.x to 4.x. Although, looking back, I would not recommend it. When downgrading your GCC, you run the risk of breaking other things in your system. I'd recommend a VM install of 14.04. Hope that solves your problem. If it doesn't still, write to me with your entire Output log, maybe we can sort it out.
